src/plugins/intel_cpu/src/nodes/rope.cpp

```diff
@@ -221,6 +619,7 @@ void RoPE::initSupportedPrimitiveDescriptors() {
 
     auto rtPrecision = srcPrecision;
     auto CosSinPrecision = ov::element::f32;
+    bool can_inplace = true;
```

Comment:
Making inplace optimization `true` by default is not very safe in context of future RoPE extension. Can we disable it by default and explicitly enable only for valid cases below?

Reply:
Done, only llama like will be enabled.